dime-coding-standards/checklists/tex/code-review-package-checklist

```diff
@@ -0,0 +1,133 @@
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{All scripts to be reviewed.}
+			\item[] \dimeCheckBox{All data necessary to run the code to be reviewed - in deidentified form.}
+			\item[] \dimeCheckBox{A README. file that contains the following information:}
+                        \begin{itemize}
+                            \item[] \dimeCheckBox{Purpose of each folder}
+                            \item[] \dimeCheckBox{Purpose of each script (if more than one script is used)}
+                            \item[] \dimeCheckBox{Is all confidential data de-identified? (if provided in the review)} 
+                             \item[] \dimeCheckBox{Outputs produced (if part of the review)}
+                             \item[] \dimeCheckBox{Which script creates which output}
+                             
+                            
+                        \end{itemize}
+			\item[] \dimeTextField{master}{11cm}{README. file path and name:}
+			\item[] \dimeCheckBox{A \textbf{master script} that runs all the necessary code and allows the reviewer to adjust file paths only once.}
+		\end{itemize}
+		
+		\vspace{.6em}
+		
+		\subsection{2. \, Master script requirements}	
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{ Master script clearly indicates which line needs to be changed to run in a different computer}
+			\item[] \dimeCheckBox{ The master script runs from start to end if the reviewer changes only directory path(s) in one location of the master script}
+			\item[] \dimeCheckBox{ The master script installs any commands required (e.g. from SSC in Stata, CRAN in R, etc.)}
+			\item[] \dimeCheckBox{ In Stata, the master do file sets critical configurations such as version, matsize, and varabbrev (either sets directly or through a wrapper command such as ieboilstart from ietoolkit}
+			\item[] \dimeCheckBox{ In R, all necessary packages to run the code are loaded in the master script}
+			\item[] \dimeTextField{time}{2.5cm}{\textbf{Approximate time} it takes to run all code from the the master script: }
+		\end{itemize}
+		
+		\subsection{2. \, Data}
+		
+		The package includes:
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Fully de-identified data. \textbf{Any exception where sensitive data is strictly needed to conduct the review will require additional ethics and security measures, and will be handled on a case-by-case basis.}}
+			\item[] \dimeCheckBox{All data necessary to run the code to be reviewed is included}
+		\end{itemize}
+		
+		
+		\subsection{3. \, Outputs}
+		
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{ All outputs created by the code to be review are included}
+			\item[] \dimeCheckBox{ All raw outputs to be reviewed are created in the same folder structure as those originally submitted.}
+			\item[] \dimeCheckBox{ It is easy to identify which code creates which output (\textbf{(either in master script or README. file)}}
```

Comment:
this is already covered in the readme -- there's a checkbox under the readme for "Which script creates which output"